addons/missileguidance/stringtable.xml

```diff
@@ -28,80 +28,8 @@
         
             /* Weapons */
-        <Key ID="STR_ACE_Hellfire_AGM114K">
-          <English>Hellfire II AGM-114K Missile</English>
-          <Spanish></Spanish>
-          <French></French>
-          <Polish></Polish>
-          <German></German>
-          <Czech>Hellfire II AGM-114K</Czech>
-          <Italian></Italian>
-          <Portuguese></Portuguese>
-          <Hungarian></Hungarian>
-          <Russian></Russian>
-        </Key>
-        <Key ID="STR_ACE_Hellfire_AGM114K_Short">
-          <English>AGM-114K</English>
-          <Spanish></Spanish>
-          <French></French>
-          <Polish></Polish>
-          <German></German>
-          <Czech>AGM-114K</Czech>
-          <Italian></Italian>
-          <Portuguese></Portuguese>
-          <Hungarian></Hungarian>
-          <Russian></Russian>
-        </Key>
-        <Key ID="STR_ACE_Hellfire_AGM114K_desc">
-          <English>Hellfire II AGM-114K Laser Guided Missile</English>
-          <Spanish></Spanish>
-          <French></French>
-          <Polish></Polish>
-          <German></German>
-          <Czech>Hellfire II AGM-114K laserem naváděná střela</Czech>
-          <Italian></Italian>
-          <Portuguese></Portuguese>
-          <Hungarian></Hungarian>
-          <Russian></Russian>
-        </Key>
-    
-  </Package>
+        <Key ID="STR_ACE_Hellfire_AGM114K"><English>Hellfire II AGM-114K Missile</English><Spanish></Spanish><French></French><Polish>Hellfire II AGM-114K</Polish><German></German><Czech>Hellfire II AGM-114K</Czech><Italian></Italian><Portuguese></Portuguese><Hungarian></Hungarian><Russian></Russian></Key><Key ID="STR_ACE_Hellfire_AGM114K_Short"><English>AGM-114K</English><Spanish></Spanish><French></French><Polish>AGM-114K</Polish><German></German><Czech>AGM-114K</Czech><Italian></Italian><Portuguese></Portuguese><Hungarian></Hungarian><Russian></Russian></Key><Key ID="STR_ACE_Hellfire_AGM114K_desc"><English>Hellfire II AGM-114K Laser Guided Missile</English><Spanish></Spanish><French></French><Polish>Laserowo naprowadzana rakieta Hellfire II AGM-114K</Polish><German></German><Czech>Hellfire II AGM-114K laserem naváděná střela</Czech><Italian></Italian><Portuguese></Portuguese><Hungarian></Hungarian><Russian></Russian></Key></Package>
```

Comment:
He's using the latest public version that uses 4 spaces for serializing and `<Package> ` gets ignored.